tests/aggregation/tests.py

```diff
@@ -2126,6 +2126,15 @@ def test_group_by_reference_subquery(self):
         qs = Publisher.objects.filter(pk__in=author_qs)
         self.assertCountEqual(qs, [self.p1, self.p2, self.p3, self.p4])
 
+    def test_group_by_formed_with_static_values(self):
```

Comment:
This doesn't fail on main (on sqlite). Do you see the same?

Reply:
I'm getting `django.db.utils.OperationalError: a GROUP BY clause is required before HAVING` with `3.37.0` but [it appears to be fixed on 3.39.4](https://www.sqlite.org/releaselog/3_39_4.html)

> The [HAVING clause](https://www.sqlite.org/lang_select.html#resultset) of a [SELECT statement](https://www.sqlite.org/lang_select.html) is now allowed on any aggregate query, even queries that do not have a [GROUP BY clause](https://www.sqlite.org/lang_select.html#resultset).

Reply:
[Dev supports 3.27.0+](https://docs.djangoproject.com/en/dev/ref/databases/#sqlite-notes) so we could make the feature/ops version based but it's still mandatory. Nice catch @jacobtylerwalls

Reply:
Yes nice catch. I inadvertently had my Django environment still on Python 3.10 for some reason.

I guess we could earmark these changes for removal once the minimum Python/SQLite version is met? 🤔